templates/templates/kyma-gke-compass-integration.yaml

```diff
@@ -73,19 +73,19 @@ presubmits: # runs on PRs
         - <<: *test_infra_ref
           base_ref: master
 
-{{- range (matchingReleases .Global.releases "1.12" nil) }}
-    - name: pre-rel{{ . | replace "." "" }}-kyma-gke-compass-integration
-      branches:
-        - release-{{ . }}
-      always_run: false
-      <<: *gke_integration_job_template_k15
-      labels:
-        preset-build-release: "true"
-        <<: *gke_integration_job_labels_template
-      extra_refs:
-        - <<: *test_infra_ref
-          base_ref: release-{{ . }}
-{{- end }}
+# {{- range (matchingReleases .Global.releases "1.12" nil) }}
+#     - name: pre-rel{{ . | replace "." "" }}-kyma-gke-compass-integration
+#       branches:
+#         - release-{{ . }}
+#       always_run: false
+#       <<: *gke_integration_job_template_k15
+#       labels:
+#         preset-build-release: "true"
+#         <<: *gke_integration_job_labels_template
+#       extra_refs:
+#         - <<: *test_infra_ref
+#           base_ref: release-{{ . }}
+# {{- end }}
```

Comment:
Please remove commented code. We will be able to find it in git history, when we want to restore it.